src/public/Interfaces.ts

```diff
@@ -34,6 +34,10 @@ export interface Context {
      * Trigger metadata and function invocation data.
      */
     bindingData: { [key: string]: any };
+    /**
+     * TraceContext which is obtained from the host. Properties on this object are generated from the RpcRequest.traceContext
+     */
+    traceContext: { [key: string]: any };
```

Comment:
I think you should still include the interface for the traceContext object, since that will always be the same. In RpcConverters, we can just say that we expect something that conforms to this interface